### PR TITLE
Cxx modules remove root maps

### PR DESCRIPTION
**Pull request by vgvassilev**

### CODE REVIEW

File: cmake/modules/RootNewMacros.cmake [hunk 1]
```diff
@@ -374,7 +374,7 @@ function(ROOT_GENERATE_DICTIONARY dictionary)
   endif()
 
   if(CMAKE_ROOTTEST_NOROOTMAP)
-    set(rootmapname )
+    set(rootmap_name )
```
[the remark this answers is not in the document]
Comment (vgvassilev) [reply cmake/modules/RootNewMacros.cmake]: This is a pre-existing bug, irrelevant to the PR.

File: core/metacling/src/TCling.cxx [hunk 2]
```diff
@@ -5240,7 +5240,7 @@ Int_t TCling::LoadLibraryMap(const char* rootmapfile)
          }
       }
       delete paths;
-      if (!fMapfile->GetTable()->GetEntries()) {
+      if (fMapfile->GetTable() && !fMapfile->GetTable()->GetEntries()) {
```
[the remark this answers is not in the document]
Comment (vgvassilev) [reply core/metacling/src/TCling.cxx]: This bug is unreachable in a regular ROOT setup.

**Comment by yamaguchi1024** [comment]
This shouldn't be merged yet as we still have problems, see #2380

**Comment by vgvassilev** [comment]
@phsft-bot build just on slc6/gcc62 with flags -Druntime_cxxmodules=On -Dctest_test_exclude_none=on -DCMAKE_BUILD_TYPE=Debug -DLLVM_BUILD_TYPE=Debug

**Comment by vgvassilev** [comment]
Seems like I cannot for some reason add commits to this PR.

**Pull request closed without merging**